src/Filter/PipeInputFilter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * MiniAsset
+ * Copyright (c) Mark Story (http://mark-story.com)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Mark Story (http://mark-story.com)
+ * @since         0.0.1
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace MiniAsset\Filter;
+
+use MiniAsset\AssetTarget;
+use MiniAsset\Filter\AssetFilter;
+use MiniAsset\Filter\CssDependencyTrait;
+
+/**
+ * Pre-processing filter that adds support for command pipes
+ */
+class PipeInputFilter extends AssetFilter
+{
+    use CssDependencyTrait {
+        getDependencies as getCssDependencies;
+    }
+
+    protected $_settings = array(
+        'ext' => '.css',
+        'command' => '/bin/cat',
+        'dependencies' => true,
+        'optional_dependency_prefix' => false,
+        'path' => '/bin',
+    );
+
+    /**
+     * It will use prefixed files if they exist.
+     *
+     * @var string
+     */
+    protected $optionalDependencyPrefix = null;
+
+    public function hasDependencies()
+    {
+        return $this->_settings['dependencies'];
+    }
+
+    public function getDependencies(AssetTarget $file)
+    {
+        if ($this->_settings['dependencies']) {
+            $this->optionalDependencyPrefix = $this->_settings['optional_dependency_prefix'];
+
+            return $this->getCssDependencies($file);
```

Comment:
How do you know that the file being processed is CSS?

Reply:
When the `dependencies` setting is true the CSS-like dependencies are used. I was previously using a text setting in case there were more dependency types added later but I've decided to simplify. It should be set to false when there's no CSS-like imports.

Reply:
Wouldn't a safer default be `false` then? In the docblock for the settings/filter the usage could be explained.

Reply:
> Wouldn't a safer default be `false` then? In the docblock for the settings/filter the usage could be explained.

I think I did it because the default extension is `.css`. Maybe I should change that too.

Reply:
@markstory, please, review the settings changes in the new commit.